diff -uNr rootfs.original/home/lava/origin_printer_data/config/printer.cfg rootfs/home/lava/origin_printer_data/config/printer.cfg
--- rootfs.original/home/lava/origin_printer_data/config/printer.cfg	2025-11-06 05:01:19.000000000 +0100
+++ rootfs/home/lava/origin_printer_data/config/printer.cfg	2025-12-04 10:31:52.832922872 +0100
@@ -1258,4 +1258,9 @@
 #*# max_x = 240.0
 #*# min_y = 20.0
 #*# max_y = 240.0
-#*#
\ No newline at end of file
+#*#
+
+# Firmware-provided configuration (overwritten on firmware update)
+[include /home/lava/klipper-config/*.cfg]
+# User configuration files (persists across firmware updates)
+[include klipper/*.cfg]
